draft/index.html

```diff
@@ -517,12 +522,26 @@ <h3>Structure of a Reconciliation Query</h3>
 	    <dt><code>limit</code></dt>
             <dd>A limit on the number of candidates to return, which must be a positive integer;</dd>
 	    <dt><code>properties</code></dt>
-	    <dd>An array of objects, where each object maps a <a href='#properties'>property</a> identifier (in the <code>pid</code> field) 
-        to one or more <a>property values</a> (in the <code>v</code> field). These are used to further filter the set of candidates (similar to a WHERE clause in SQL),
-		by allowing clients to specify other attributes of entities that should match, beyond their name in the <code>query</code> field.
-		How reconciliation services handle this further restriction ("must match all properties" or "should match some") and how it affects the score, is up to the service.
-		A reconciliation service that supports properties SHOULD provide a <a>suggest service</a> for discovering these properties;</dd>
-          </dl>
+        <dd><p>An array of objects, where each object maps a <a href='#properties'>property</a> identifier (in the <code>pid</code> field) to one or more <a>property values</a> (in the <code>v</code> field).
+        These are used to further refine the list of candidates by allowing clients to specify other attributes of entities, beyond their name in the <code>query</code> field.
+        A reconciliation service that supports properties SHOULD provide a <a>suggest service</a> for discovering these properties.</p>
+        <p>In addition to <code>pid</code> and <code>v</code>, services MAY support the following optional fields that allow clients to specify the effect of each property on the resulting list of candidates.
+          If these fields are omitted, the exact behavior ("must match all", "should match some", etc.) is up to the service.</p>
+        <dl>
+          <dt><code>required</code></dt>
```

Comment:
I recognize that renaming things is one of the pounds of flesh required by the standardization process, but hopefully the revision notes will include a mapping from the old (`match=`) to the new (`required=`).

Reply:
I think what @fsteeg meant by "renaming" is a renaming from the other solutions proposed in the corresponding issues tackled by this PR, not from any parameter currently existing in the specs. At least I am not aware where this `mode` parameter would currently be used in the specs (even in the `draft` version).

Reply:
Yes, sorry, my wording was confusing here (Antonin understood it the same way in our meeting yesterday). This PR only adds 3 new, optional fields.

Reply:
Yes, I was confused. It looks like the old syntax was actually `filter=all,any,not,should`, along with `exact=true,false` https://developers.google.com/freebase/v1/search but since there's been such a big gap in time between implementations, it's probably not a big deal.